base/special/j0f.jl

```diff
@@ -0,0 +1,324 @@
+# e_j0f.c -- float version of e_j0.c.
+# Conversion to float by Ian Lance Taylor, Cygnus Support, ian@cygnus.com.
+
+# ====================================================
+# Copyright (C) 1993 by Sun Microsystems, Inc. All rights reserved.
```

Comment:
would need to be added to license.md and the exception list on contrib/add_license_to_files.jl